### PR TITLE
revert back to whole object for distributors endpoint

### PR DESCRIPTION
**Pull request by Lonestarrk**
Some internal discussion with @ricsv has led to this.. I hope no one has any issues with it.

### CODE REVIEW

File: OpenAPI Specification.json [hunk 1]
```diff
@@ -355,7 +355,7 @@
           "distributors": {
             "type": "array",
             "items": {
-              "$ref": "#/components/schemas/DistributorSummary"
```
Comment (ricsv) [inline OpenAPI Specification.json:358]: DistributorSummary should be removed if not used after this change

Comment (Lonestarrk) [reply OpenAPI Specification.json:358]: its used in the meal response

Comment (wincentek) [reply OpenAPI Specification.json:358]: Sure, go ahead. Make the change.

**Review by wincentek — APPROVED** [review]
Ok, We've not implemented this API yet and are not currently affected by the change, so no problem from our end.

**Review by bjornwiberg — APPROVED** [review]
LGTM

File: OpenAPI Specification.json (hunk 1 above)
Comment (Lonestarrk) [reply OpenAPI Specification.json:358]: @ricsv would you complete the PR

**Review by ricsv — APPROVED** [review]

**Pull request merged**